.eslintrc.json

```diff
@@ -8,24 +8,19 @@
     "es6": true
   },
   "plugins": [
-    "node",
+    "n",
     "unicorn"
   ],
   "extends": [
     "eslint:all",
+    "plugin:n/recommended",
```

Comment:
There is no "all" for this plugin, but in addition to the explicit rules below, I thought opting into the `recommended` would be helpful for any future updates

Reply:
Good idea!